test/plugins/cpuUsageThrottle.test.js

```diff
@@ -30,7 +30,7 @@ describe('cpuUsageThrottle', function () {
         var opts = { limit: 0, interval: 500 };
         var plugin = cpuUsageThrottle(opts);
         function next (cont) {
-            clearTimeout(plugin._timeout);
+            plugin.close();
```

Comment:
Should this just be in an `afterEach` block?

Reply:
Is there a way to do that without maintaining state outside the test?

Reply:
Not that I know of - usually just use a `plugin` var within the closure.